README.rst

```diff
@@ -0,0 +1,281 @@
+tntcxx
```

Comment:
The header is quite meaningless in the context of our documentation website. Let's give it a proper title.